jekyll/_cci2/api-developers-guide.md

```diff
@@ -66,26 +65,26 @@ It is recommended that you specify an Accept header in your API requests. The ma
 of API endpoints will return JSON by default, but some endpoints (primarily API
 v1) return EDN if no accept header is specified.
 
-To return formatted JSON, include a `text/plain` header like the example shown below:
+- To return formatted JSON, include a `text/plain` header like the example shown below:
 
-```shell
-curl --header "Circle-Token: $CIRCLE_TOKEN" \
-  --header 'Accept: text/plain'    \
-  https://circleci.com/api/v2/project/{project-slug}/pipeline
-```
+  ```shell
+  curl --header "Circle-Token: $CIRCLE_TOKEN" \
+    --header 'Accept: text/plain'    \
+    https://circleci.com/api/v2/project/{project-slug}/pipeline
+  ```
 
-To return compressed JSON:
+- To return compressed JSON:
 
-```shell
-curl --header "Circle-Token: $CIRCLE_TOKEN" \
-  --header 'Accept: application/json'    \
-  https://circleci.com/api/v2/project/{project-slug}/pipeline
-```
+  ```shell
+  curl --header "Circle-Token: $CIRCLE_TOKEN" \
+    --header 'Accept: application/json'    \
+    https://circleci.com/api/v2/project/{project-slug}/pipeline
+  ```
 
 ## Getting started with the API
 {: #getting-started-with-the-api }
 
-The CircleCI API shares similarities with previous API versions in that it identifies your projects using repository name. For instance, if you want to pull information from CircleCI about the GitHub repository https://github.com/CircleCI-Public/circleci-cli you can refer to that in the CircleCI API as `gh/CircleCI-Public/circleci-cli`, which is a "triplet" of the project type (VCS provider), the name of your "organization" (or your username), and the name of the repository.
+The CircleCI API shares similarities with previous API versions in that it identifies your projects using repository name. For instance, if you want to pull information from CircleCI about the GitHub repository "https://github.com/CircleCI-Public/circleci-cli" you can refer to that in the CircleCI API as `gh/CircleCI-Public/circleci-cli`, which is a "triplet" of the project type (VCS provider), the name of your "organization" (or your username), and the name of the repository.
```

Comment:
We don't use quotes very often and I'm unsure if they fit here either. Maybe change "organization" to be more specific about the role and put it in bold like we do elsewhere when we reference things about roles? I'm also not sure the link belongs in a quote. Will the link still be clickable in quotes? If so, I think the quotes will make it seem more crowded. If the quotes escape the link and it's not clickable, then I do think they make sense in that instance.